src/tasks/actions/thunks.ts

```diff
@@ -558,6 +558,25 @@ export const createTaskFromTemplate = (template: TaskTemplate) => async (
   }
 }
 
+export const retryTask = (taskID: string, runID: string) => async (
+  dispatch: Dispatch<Action | NotificationAction>
+) => {
+  try {
+    const resp = await api.postTasksRunsRetry({
+      taskID,
+      runID,
+    })
+
+    if (resp.status !== 200) {
+      throw new Error(resp.data.message)
+    }
+
+    dispatch(notify(copy.taskRetrySuccess()))
+  } catch (error) {
+    console.error(error)
```

Comment:
since it seems like notifications are sent from this thunk, let's add an error notification here, rather than doing a `console.error` if there's a helpful error message returned, we'll want to show the user it.